scss/_close.scss

```diff
@@ -47,3 +47,11 @@
 .btn-close-white {
   filter: var(--#{$prefix}btn-close-white-filter);
 }
+
+@if $enable-dark-mode {
+  [data-bs-theme="dark"] {
```

Comment:
Can't button close white variant be treated like the `.carousel-dark`?

Something like:

```scss
%btn-close-white {
  filter: var(--#{$prefix}btn-close-white-filter);
}

.btn-close-white {
  @extend %btn-close-white;
}

@if $enable-dark-mode {
  [data-bs-theme="dark"] {
    .btn-close {
      @extend %btn-close-white;
    }
  }
}
```

and so a deprecated mention of this `.btn-close-white` class in the docs